### ISSUE TITLE
Issue installing grenier

### ISSUE THREAD

**Issue by ihodes** [let-def/grenier#2]
via `opam install`

```
=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫
[ERROR] The compilation of grenier failed at "make install".
Processing  1/1: [grenier: ocamlfind remove]
#=== ERROR while installing grenier.0.1 =======================================#
# opam-version 1.2.2
# os           darwin
# command      make install
# path         /Users/isaachodes/.opam/system/build/grenier.0.1
# compiler     system (4.02.3)
# exit-code    2
# env-file     /Users/isaachodes/.opam/system/build/grenier.0.1/grenier-68931-ad8886.env
# stdout-file  /Users/isaachodes/.opam/system/build/grenier.0.1/grenier-68931-ad8886.out
# stderr-file  /Users/isaachodes/.opam/system/build/grenier.0.1/grenier-68931-ad8886.err
### stdout ###
# [...]
# make[2]: `doubledouble.cma' is up to date.
# make[2]: `doubledouble.cmxa' is up to date.
# make[1]: Nothing to be done for `all'.
# make[2]: `baltree.cma' is up to date.
# make[2]: `baltree.cmxa' is up to date.
# make[2]: `trope.cma' is up to date.
# make[2]: `trope.cmxa' is up to date.
#
# Installing library with ocamlfind
# ocamlfind install  grenier META hll/hll.mli hll/hll.cmi hll/hll_consts.cmi hll/hll.a hll/hll.cma hll/hll.cmxa jmphash/jmphash.mli jmphash/jmphash.cmi jmphash/jmphash.a jmphash/jmphash.cma jmphash/jmphash.cmxa pcg/pcg.mli pcg/pcg.cmi pcg/pcg.a pcg/pcg.cma pcg/pcg.cmxa orderme/order_list.mli orderme/order_indir.mli orderme/order_managed.mli orderme/order_list.cmi orderme/order_indir.cmi orderme/order_managed.cmi orderme/orderme.a orderme/orderme.cma orderme/orderme.cmxa orderme/dllorderme_stubs.so orderme/liborderme_stubs.a doubledouble/doubledouble.mli doubledouble/doubledouble.cmi doubledouble/doubledouble.cmo doubledouble/doubledouble.cmx physh/physh.mli physh/physh.cmi physh/physh.a physh/physh.cma physh/physh.cmxa physh/lib_physh_stubs.a physh/dll_physh_stubs.so baltree/baltree.a baltree/baltree.cma baltree/baltree.cmxa baltree/bt1.mli baltree/bt1.cmi baltree/bt2.mli baltree/bt2.cmi baltree/mbt.mli baltree/mbt.cmi trope/trope.a trope/trope.cma trope/trope.cmxa trope/trope.mli trope/trope.cmi
### stderr ###
# [...]
# Installed /Users/isaachodes/.opam/system/lib/grenier/physh.cma
# Installed /Users/isaachodes/.opam/system/lib/grenier/physh.a
# Installed /Users/isaachodes/.opam/system/lib/grenier/physh.cmi
# Installed /Users/isaachodes/.opam/system/lib/grenier/physh.mli
# Installed /Users/isaachodes/.opam/system/lib/grenier/doubledouble.cmx
# Installed /Users/isaachodes/.opam/system/lib/grenier/doubledouble.cmo
# Installed /Users/isaachodes/.opam/system/lib/grenier/doubledouble.cmi
# Installed /Users/isaachodes/.opam/system/lib/grenier/doubledouble.mli
# ocamlfind: orderme/liborderme_stubs.a: No such file or directory
# make: *** [libinstall] Error 2
```

**Comment by rgrinberg** [comment]
I get the same error

**Comment by def-lkb** [comment]
Thanks, this should be fixed. Sorry for that.
I assume that you found this looking for latest merlin. `grenier` & `sturgeon` are used only as development tools, the dependency will become optional for normal uses.

**Comment by ihodes** [comment]
No worries; I was trying to build merlin for some development.

**Comment by let-def** [comment]
Fixed by b443c69e7ed042424b55017511e2f847ff776b65
